diagnostic.md

```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The router determines what view a url corresponds to, and connects to the resources that should be called upon to render that view. A route either holds sample data or is the place to put database requests.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.